integration/examples/remote-multi-config-microservices/README.md

```diff
@@ -0,0 +1,50 @@
+### Example: Remote config µSvcs with Skaffold
+
+In this example:
+
+* Deploy microservice applications from a remote git repository using Skaffold.
+
+Skaffold can build and deploy from configurations defined in remote git repositories. In this example, we'll walk through using skaffold to deploy two applications, an exposed "web" frontend which calls an unexposed "app" backend from the [examples/multi-config-microservices](../multi-config-microservices) project as a remote dependency.
+
+**WARNING: If you're running this on a cloud cluster, this example will create a service and expose a webserver.
+It's highly suggested that you only run this example on a local, private cluster like minikube or Kubernetes in Docker for Desktop.**
+
+#### Running the example on minikube
+
+From this directory, run:
+
+```bash
+skaffold dev
+```
+
+Now, in a different terminal, hit the `leeroy-web` endpoint
+
+```bash
+$ curl $(minikube service leeroy-web --url)
+leeroooooy app!
+```
```

Comment:
the `leeroy-web` service doesn't actually have a service resource defined for it, so this part fails. Not sure if we want to add that in this PR or a separate one.

I tried adding a service to this and it seems like the url output by this minikube command doesn't work. Not sure if that's just an issue on my end tho

Reply:
wow, I checked the history on the `microservices` example and `curl $(minikube service leeroy-web --url)` has been on it since the beginning. However, you're right, this command doesn't work. Maybe the behavior changed on `minikube` sometime in the past. Anyways I've modified this example. Will send a separate PR to fix for the other examples.